src/components/Problems/Problem.tsx

```diff
@@ -151,7 +151,16 @@ export const Problem: FC<{
             diskusia ({problem.num_comments})
           </Button>
           {canSubmit && (
-            <Button onClick={handleUploadClick} disabled={isAuthed && !canRegister} variant="button2">
+            <Button
+              onClick={handleUploadClick}
+              disabled={isAuthed && !canRegister}
+              disabledTooltip={
+                isAuthed
+                  ? 'Tejto súťaže sa nemôžeš zúčastniť, pretože si vo vysokom ročníku. Ak ho máš nastavený zle, zmeň si ho v profile.'
+                  : 'Nie si prihlásený'
```

Comment:
ak nie si prihlaseny tak bude ovela lepsie keby ti to rovno otvorilo prihlasovaci dialog namiesto toho aby to bolo disablnute